std/testing/asserts_test.ts

```diff
@@ -126,7 +126,7 @@ Deno.test("testingEqual", function (): void {
 Deno.test("testingNotEquals", function (): void {
   const a = { foo: "bar" };
   const b = { bar: "foo" };
-  assertNotEquals(a, b);
+  assertNotEquals<unknown>(a, b);
```

Comment:
The unknown casting in this file is only necessary because we are comparing values that obviously has different types.